docs/UserGuide.adoc

```diff
@@ -227,6 +227,35 @@ Format: `exit`
 Address book data are saved in the hard disk automatically after any command that changes the data. +
 There is no need to save manually.
 
+== Features(Coming in v2.0)
+
+=== Add relationship
+
+Enters the indices of the two people in between whom the user wants to add an relationship +
+as seen in the last shown list. +
+Format: `Add Relationship NAME1 NAME2`
+
+=== Add named relationship between people
+
+Enters the indices of the two people in between whom the user wants to add an relationship +
+as seen in the last shown list and the name of the relationship. +
+If the relationship between these two people is not in the address book, one new relationship will be created upon +
+entering this command and given a name. +
+The order of the names of people entered does not matter.
```

Comment:
Same problems plaguing this part as the ones in the previous segment: (i) a description that explains the usage of the command instead and (ii) inconsistency in the suggested usage

Furthermore, because of the duplicity in the functionalities of this command and the abovementioned command, I think you may want to consider collapsing the two into `Add named relationship between people` such that if no name for the relationship is given as an argument by the user, a new unnamed relationship will simply be created